roles/ansible-test-inventory/templates/inventory-nxos.j2

```diff
@@ -7,21 +7,22 @@
 nxos-9 ansible_host={{ hostvars['nxos'].ansible_host }} ansible_user=zuul
 
 [nxos:vars]
-ansible_connection=network_cli
+ansible_connection={% if ansible_test_collections is defined -%}ansible.netcommon.{%- endif %}network_cli
+ansible_network_os={% if ansible_test_collections is defined -%}cisco.nxos.{%- endif %}nxos
```

Comment:
This makes sure that the cliconf from the collection is pulled and not ansible/ansible one when run with stable-2.9.